src/Arcus.Security.Tests.Integration/HashiCorp/Hosting/HashiCorpVaultTestServer.cs

```diff
@@ -84,10 +95,14 @@ private HashiCorpVaultTestServer(Process process, string rootToken, string liste
         /// <exception cref="ArgumentNullException">Thrown when the <paramref name="configuration"/> or <paramref name="logger"/> is <c>null</c>.</exception>
         public static async Task<HashiCorpVaultTestServer> StartServerAsync(TestConfig configuration, ILogger logger)
         {
-            Guard.NotNull(logger, nameof(logger),
-                "Requires a logger for logging diagnostic trace messages during the lifetime of the test server");
-            Guard.NotNull(configuration, nameof(configuration),
-                "Requires a configuration instance to retrieve the HashiCorp Vault installation folder");
+            if (logger is null)
+            {
+                throw new ArgumentNullException(nameof(logger), "Requires a logger for logging diagnostic trace messages during the lifetime of the test server");
+            }
+            if (configuration is null)
```

Comment:
```suggestion

            if (configuration is null)
```
Best to have a blank line between curly brackets, following the Microsoft code style.